infra/nnfw/cmake/packages/TensorFlowLite-2.8.0/TensorFlowLiteConfig.cmake

```diff
@@ -8,7 +8,7 @@ if(BUILD_TENSORFLOW_LITE)
   macro(return_unless VAR)
   if(NOT ${VAR})
     message("TFLite 2.8: ${VAR} NOT TRUE")
-    set(TensorFlowLite_FOUND FALSE PARENT_SCOPE)
+    set(TensorFlowLite_FOUND FALSE)
```

Comment:
As I understand `PARENT_SCOPE` is for caller of this file not for the macro.
If removed, `TensorFlowLite_FOUND` is not set for parent scope.
Maybe above in line 3 is should have `PARENT_SCOPE`
Anyway I am not 100% confident with my comments -_-;